static/app/components/searchSyntax/parser.tsx

```diff
@@ -415,6 +416,14 @@ export class TokenConverter {
     };
   };
 
+  tokenParen = (value: '(' | ')') => ({
+    ...this.defaultTokenFields,
+    value,
+    left: value === '(',
+    right: value === ')',
+    type: Token.PAREN as const,
```

Comment:
Small nit, usually the type comes right after `defaultTokenFields`

```suggestion
    ...this.defaultTokenFields,
    type: Token.PAREN as const,
    value,
    left: value === '(',
    right: value === ')',
```